test/tests/utilitiesTest.js

```diff
@@ -585,4 +585,35 @@ describe("Zotero.Utilities", function() {
 			});
 		});
 	});
+
+
+	describe("cleanEdition", function() {
+		it('should clean valid editions (allowOtherText == false)', function() {
+			for (let [text, expected] of [
+				['2. Auflage', '2'],
+				['1. Auflage', null],
```

Comment:
Should this be an empty string instead? Either probably works, but technically if we're setting this directly do a field it should probably be an empty string.